testsuite/run_sets/secondary.yml

```diff
@@ -58,5 +58,8 @@
 - features/secondary/min_change_software_channel.feature
 - features/secondary/min_retracted_patches.feature
 - features/secondary/proxy_cobbler_pxeboot.feature
-
+- features/secondary/min_move_from_proxy.feature
+- features/secondary/min_move_to_proxy.feature
```

Comment:
I see that as a unique feature, I think that for maintenance and for organization of features and their strict order, it will be better to join these two in one.

Reply:
From a conceptual point of view both split in one or two features make sense to me.

BUT

From a practical point of view we can probably remove cleanup and preparation steps and therefore win execution time if we merge into one feature.

So yes, I think Oscar's suggestion makes sense.

Reply:
That's what I tried first, but then the same events appeared twice and "I wait until event" on the second event incorrectly matched the first one. Is there any workaround for this?